app/views/view-application.njk

```diff
@@ -26,30 +27,30 @@
       {% endset -%}
 
       {% set vetReviewHtml %}
-        {% if vetVisit %}
-          <h2 class="govuk-heading-l">Data inputted</h2>
-          {{ govukTable(vetVisitData) }}
-        {% else %}
-          <h2 class="govuk-heading-l">Not yet reviewed</h2>
-        {% endif %}
+      {% if vetVisit %}
+        <h2 class="govuk-heading-l">Data inputted</h2>
+        {{ govukTable(vetVisitData) }}
+      {% else %}
+        <h2 class="govuk-heading-l">Not yet reviewed</h2>
+      {% endif %}
       {% endset -%}
 
       {% set claimHtml %}
-        {% if claimed %}
-          <h2 class="govuk-heading-l">Selected for fraud check</h2>
-          {{ govukTable(claimData) }}
-        {% else %}
-          <h2 class="govuk-heading-l">Not yet able to claim</h2>
-        {% endif %}
+      {% if claimed %}
+        <h2 class="govuk-heading-l">Selected for fraud check</h2>
+        {{ govukTable(claimData) }}
+      {% else %}
+        <h2 class="govuk-heading-l">Not yet able to claim</h2>
+      {% endif %}
       {% endset -%}
 
       {% set paymentHtml %}
-        {% if payment %}
-          <h2 class="govuk-heading-l">Payment information</h2>
-          {{ govukTable(paymentData) }}
-        {% else %}
-          <h2 class="govuk-heading-l">Not yet paid</h2>
-        {% endif %}
+      {% if payment %}
+        <h2 class="govuk-heading-l">Payment information</h2>
+        {{ govukTable(paymentData) }}
+      {% else %}
+        <h2 class="govuk-heading-l">Not yet paid</h2>
+      {% endif %}
       {% endset -%}
 
       {{ govukTabs({
```

Comment:
Could this be a model which is part of the added to the view in the route? This would make the .njk file a lot neater and bloated.